nohup: ignoring input
Total queries read: 40
Splitting queries:   0%|          | 0/1 [00:00<?, ?it/s]Splitting queries: 100%|██████████| 1/1 [00:00<00:00, 7884.03it/s]
Fetching schema:   0%|          | 0/1 [00:00<?, ?it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Fetching schema: 100%|██████████| 1/1 [00:00<00:00,  3.23it/s]Fetching schema: 100%|██████████| 1/1 [00:00<00:00,  3.23it/s]
Processing queries:   0%|          | 0/40 [00:00<?, ?it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:   5%|▌         | 2/40 [00:00<00:02, 18.51it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  10%|█         | 4/40 [00:00<00:01, 19.13it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  15%|█▌        | 6/40 [00:00<00:01, 18.36it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  20%|██        | 8/40 [00:00<00:01, 18.25it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  25%|██▌       | 10/40 [00:00<00:01, 17.98it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  30%|███       | 12/40 [00:00<00:01, 16.81it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  35%|███▌      | 14/40 [00:00<00:01, 16.88it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  40%|████      | 16/40 [00:00<00:01, 17.23it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  45%|████▌     | 18/40 [00:01<00:01, 17.13it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  50%|█████     | 20/40 [00:01<00:01, 17.34it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  55%|█████▌    | 22/40 [00:01<00:01, 16.99it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  60%|██████    | 24/40 [00:01<00:00, 17.17it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  65%|██████▌   | 26/40 [00:01<00:00, 17.18it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  70%|███████   | 28/40 [00:01<00:00, 17.11it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  75%|███████▌  | 30/40 [00:01<00:00, 17.12it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  80%|████████  | 32/40 [00:01<00:00, 17.26it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  85%|████████▌ | 34/40 [00:01<00:00, 17.04it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  90%|█████████ | 36/40 [00:02<00:00, 17.31it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries:  95%|█████████▌| 38/40 [00:02<00:00, 17.12it/s]/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
/data/hdd1/users/kmparmp/Neo/sql_parser/config.py:92: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)
Processing queries: 100%|██████████| 40/40 [00:02<00:00, 17.06it/s]Processing queries: 100%|██████████| 40/40 [00:02<00:00, 17.29it/s]
40
0
  0%|          | 0/40 [00:00<?, ?it/s]  5%|▌         | 2/40 [00:00<00:02, 14.73it/s] 15%|█▌        | 6/40 [00:00<00:01, 26.77it/s] 22%|██▎       | 9/40 [00:00<00:01, 16.99it/s] 45%|████▌     | 18/40 [00:00<00:00, 34.75it/s] 57%|█████▊    | 23/40 [00:00<00:00, 31.68it/s] 75%|███████▌  | 30/40 [00:00<00:00, 38.55it/s] 88%|████████▊ | 35/40 [00:01<00:00, 40.58it/s]100%|██████████| 40/40 [00:01<00:00, 37.74it/s]
Total files to process: 0
Processing plans: |          | 0/0 [00:00<?, ?file/s]Processing plans: |          | 0/0 [00:00<?, ?file/s]

Finished processing: 0 plans | Skipped: 0 test files
Saving experience to file...
INFO:algorithm.neo:Update started, step: 0, episode: 0, time: Mon Apr 28 20:21:06 2025
INFO:algorithm.neo:Start training: Mon Apr 28 20:21:06 2025
Traceback (most recent call last):
  File "/data/hdd1/users/kmparmp/Neo/run_experiment.py", line 157, in <module>
    alg.run()
  File "/data/hdd1/users/kmparmp/Neo/algorithm/neo.py", line 374, in run
    self.update_process(current_episode)
  File "/data/hdd1/users/kmparmp/Neo/algorithm/neo.py", line 476, in update_process
    losses = self.agent.train_net(
  File "/data/hdd1/users/kmparmp/Neo/algorithm/neo.py", line 275, in train_net
    train_dl = torch.utils.data.DataLoader(
  File "/data/hdd1/users/kmparmp/miniconda3/envs/neo/lib/python3.9/site-packages/torch/utils/data/dataloader.py", line 376, in __init__
    sampler = RandomSampler(dataset, generator=generator)  # type: ignore[arg-type]
  File "/data/hdd1/users/kmparmp/miniconda3/envs/neo/lib/python3.9/site-packages/torch/utils/data/sampler.py", line 164, in __init__
    raise ValueError(
ValueError: num_samples should be a positive integer value, but got num_samples=0
[W429 12:21:30.623485377 CudaIPCTypes.cpp:16] Producer process has been terminated before all shared CUDA tensors released. See Note [Sharing CUDA tensors]
